documentation/asciidoc/topics/proc_enabling_encryption.adoc

```diff
@@ -27,7 +27,7 @@ include::yaml/tls_secretname.yaml[]
 include::yaml/transport_encryption.yaml[]
 ----
 <1> Configures the transport stack to use the specified security-realm to provide cluster encryption.
-<2> Configure the keystore path in the transport realm. The secret is mounted at `/etc/encrypt/transport`.
+<2> Configure the keystore path in the transport realm. The secret is mounted at `/etc/encrypt/endpoint`.
```

Comment:
This line was actually correct, it's the example that has it wrong: https://github.com/infinispan/infinispan-helm-charts/blob/main/documentation/asciidoc/topics/yaml/transport_encryption.yaml#L15-L20

Reply:
Thanks, fixed!